lib/peep.rb

```diff
@@ -0,0 +1,34 @@
+require 'PG'
+
+class Peep
+  attr_reader :peep_id, :peep, :user, :timestamp
+
+  def initialize(id:, peep:, user:, timestamp:)
+    @peep_id = id
```

Comment:
I'm slightly unclear on what the \peep_id\ is and when/how it is generated? If it's an id which is used purely for indexing within the \peeps\ table then wouldn't it be best for it to be automatically generated by the table itself?